pages/hackathon-grant.js

```diff
@@ -0,0 +1,221 @@
+import { Box, Container, Flex, Grid, Heading } from 'theme-ui'
+import Meta from '@hackclub/meta'
+import Head from 'next/head'
+import ForceTheme from '../components/force-theme'
+import Nav from '../components/nav'
+import Footer from '../components/footer'
+import Sparkles from '../components/sparkles'
+import NextLink from 'next/link'
+import { Link, Text, Button, Card } from 'theme-ui'
+import Icon from '@hackclub/icons'
+import Form from '../components/hackathon-grant/form'
+
+const Requirement = ({ title, children }) => {
+  return (
+    <Card
+      variant="primary"
+      sx={{
+        backgroundColor: 'darkless',
+        color: 'snow',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'start'
+      }}
+    >
+      <Flex sx={{ alignItems: 'center' }}>
```

Comment:
```suggestion
      <Flex sx={{ alignItems: 'start' }}>
```

Reply:
<img width="609" alt="Screen Shot 2022-08-15 at 12 35 03 PM" src="https://user-images.githubusercontent.com/28282096/184686146-f48db67d-f5ed-4be9-8391-2a853f8f61a1.png">
would look like this ^^

instead of this (below)
<img width="547" alt="Screen Shot 2022-08-15 at 12 35 41 PM" src="https://user-images.githubusercontent.com/28282096/184686246-1aec561f-435e-47c3-9494-6950388dd666.png">